doc/content/examples/example3.md

```diff
@@ -87,4 +87,15 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+This example can also be run using the Automatic Differentiation (AD). Automatic Differentiation
+computes the "exact" empirical Jacobian of the stiffness matrix as opposed to the standard
+theoretical Jacobian.  More information can be found in
+[ISoil material](source/materials/ADComputeISoilStress.md). The plot below
+ presents the accelerations at the top soil computed using AD and no-AD.
+
+!media media/examples/AD_noAD_Comp.png
```

Comment:
Please add a plot comparing the response spectra as well. Use the responsespectracalculator VPP. If ex03 doesnt have it, please add it to that as well.